src/test/java/org/jenkinsci/plugins/ewm/steps/DiskAllocationStrategyTest.java

```diff
@@ -0,0 +1,87 @@
+package org.jenkinsci.plugins.ewm.steps;
+
+import hudson.model.Result;
+import org.jenkinsci.plugins.ewm.TestUtil;
+import org.jenkinsci.plugins.ewm.definitions.Disk;
+import org.jenkinsci.plugins.ewm.definitions.DiskPool;
+import org.jenkinsci.plugins.ewm.providers.UserProvidedDiskInfo;
+import org.jenkinsci.plugins.ewm.strategies.FastestReadSpeedStrategy;
+import org.jenkinsci.plugins.ewm.strategies.FastestWriteSpeedStrategy;
+import org.jenkinsci.plugins.workflow.job.WorkflowRun;
+import org.junit.After;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.jvnet.hudson.test.BuildWatcher;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import static java.lang.String.format;
+
+/**
+ * Tests for the Disk Allocation Strategy feature used in the Pipeline context.
+ *
+ * @author Alexandru Somai
+ */
+public class DiskAllocationStrategyTest {
+
+    @ClassRule
+    public static JenkinsRule j = new JenkinsRule();
+
+    @ClassRule
+    public static BuildWatcher watcher = new BuildWatcher();
+
+    @After
+    public void tearDown() {
+        TestUtil.removeDiskPools(j.jenkins);
+    }
+
+    @Test
+    public void useAllocationStrategyInPipeline() throws Exception {
+        Disk disk1 = TestUtil.createDisk(new UserProvidedDiskInfo(0D, 1D));
+        Disk disk2 = TestUtil.createDisk(new UserProvidedDiskInfo(0D, 3D));
+        DiskPool diskPool = TestUtil.createDiskPool(disk1, disk2);
+        TestUtil.setUpDiskPools(j.jenkins, diskPool);
+
+        WorkflowRun run = TestUtil.createWorkflowJobAndRun(j.jenkins, format("" +
+                "exwsAllocate diskPoolId: '%s', strategy: [$class: 'FastestWriteSpeedStrategy']", diskPool.getDiskPoolId()));
```

Comment:
If I want to simplify this to: 
`exwsAllocate diskPoolId: '%s', strategy: fastetWriteSpeed()` 
I need `workflow-cps` version 2.10 as test dependency.

With `workflow-cps` version 2.10, I get the following error: https://gist.github.com/alexsomai/024a8d9d911ceede13e51590ac6e1b15
Currently I haven't found a solution for it, so that's why I've used `$class` in test.